crates/astria-sequencer-relayer/tests/blackbox/helpers/test_sequencer_relayer.rs

```diff
@@ -9,6 +9,7 @@ use std::{
     io::Write,
     mem,
     net::SocketAddr,
+    str::FromStr,
```

Comment:
We can avoid this by using `parse` below

Reply:
Done in [458f2a2](https://github.com/astriaorg/astria/pull/1063/commits/458f2a27ff3a3fe97880a3b049c4359c1078058f).